tests/mockito/junit4-agent/src/test/java/samples/powermockito/junit4/agent/MockFinalUsingAnnotationsTest.java

```diff
@@ -40,7 +40,7 @@
     private FinalDemo usingMockitoMockAnnotation;
 
     @SuppressWarnings("deprecation")
-    @org.mockito.MockitoAnnotations.Mock
+    @org.mockito.Mock
     private FinalDemo usingDeprecatedMockitoMockAnnotation;
```

Comment:
by changing from the inner class to the public class, the suppression and the field name becomes outdated and pointless, the field just above `usingMockitoMockAnnotation` will be exactly the same, sans the FQCN.